EnglishTranslation-master/QUEST_LV_0100.tsv

```diff
@@ -12237,8 +12237,8 @@ QUEST_LV_0100_20151224_012237	Bishop Urbonas is the pillar sustaining the people
 QUEST_LV_0100_20151224_012238	I've been asking around to try to figure out what happened but it seems as if the priests have all dissappeared as well.{nl}
 QUEST_LV_0100_20151224_012239	My men are all looking for the whereabouts of Bishop Urbonas.. {nl}But it's pretty much a dead-end at this point.{nl}
 QUEST_LV_0100_20151224_012240	If you're as good as the rumors make you out to be, you'll have no trouble in finding where Bishop Urbonas is.{nl}Just make sure nobody knows about this.. The lord doesn't want the people being any more agitated than they already are.
-QUEST_LV_0100_20151224_012241	Could you ask that person how much longer he is planning on making us stand here?
-QUEST_LV_0100_20151224_012242	It's been days since we've arrived here, and yet we still haven't set foot in Orsha.{nl}Why did they make us travel all this way just to make us stand around like this?
+QUEST_LV_0100_20151224_012241	Could you ask that person how much longer is he planning on making us stand here?
```

Comment:
i feel like this would be correct if the question was directed at the character:
`How much longer is he planning on making us stand here?`
or if the person speaking was making you speak specifically that question to the person unrelated:
`Could you ask that person "How much longer is he planning on making us stand here?"`
but the question is actually whether or not the character can ask "that person" or not

im sorry if this isn't really making sense, but to me this sounds off 😅